src/Monarch/VirtualDom/VirtualDomTree.ts

```diff
@@ -62,6 +62,7 @@ export namespace VirtualDomTree {
     export interface ElementNS<message> extends Tagged<typeof ElementNS>, Parent<message> {
         ns?: NS
         tagName: TagName
+        key?: string
```

Comment:
It's fine to just have the `key` property inside `facts` as users give it to us. There is no need to spread it inside `ElementNS` type too.